README.md

```diff
@@ -49,6 +49,7 @@ To set a global config you have to set the value in your config.js file inside t
 | `title` | `'Olympic Winter Games 2022'` | The title above the medal table. |
 | `reloadInterval` | `1800000` (30 mins) | How often should the data be fetched. |
 | `provider` | `'bloomberg'` | Specify the data source. Possible options: `'bloomberg'` and `'nbc'`. |
+| `countryList` | `[ "NO", "AU", "US", "DE", "FR", "CA" ]` | Specify a list of country codes to display (alpha-2 code). |
```

Comment:
The default of this value is undefined. However, I think your example is valuable and should be moved inside the description of the config option.

I also think the `maxRows` option should be ignored if the `countryList` is specified.

Reply:
Awesome. Will take a look at these and submit a new PR